pkg/proc/proc_test.go

```diff
@@ -3262,9 +3229,6 @@ func frameInFile(frame proc.Stackframe, file string) bool {
 }
 
 func TestCgoStacktrace(t *testing.T) {
-	if runtime.GOARCH != "amd64" {
```

Comment:
This test is failing, if you aren't going to fix it (it's probably going to be fairly complicated) then you should keep it disabled.

Reply:
To my honest that my code is on the base of go1.10, which works well done on CgoStacktrace... And i confirmed than golang changed some asm-code in `asmcgocall` and it's call-chain... That's why it needs some some more time on cgo, and it may hard to compatible low go version(which we using now).

Reply:
Ok.